servers/rendering/shader_language.cpp

```diff
@@ -394,6 +394,9 @@ const ShaderLanguage::KeyWord ShaderLanguage::keyword_list[] = {
 	{ TK_ERROR, nullptr, CF_UNSPECIFIED, {}, {} }
 };
 
+/**
+ * Parse the next token from the code and advance the cursor.
+ */
```

Comment:
```suggestion
// Parse the next token from the code and advance the cursor.
```

Reply:
There is precedent around the codebase to do e.g.

```
/** This method uses a signed argument for better error reporting as it's used from the scripting API. */
```

so I'm not sure if there is a consistent style guide recommendation to do this instead.

Reply:
We essentially never use this style, and it's not really a doc-comment, but it's permitted in the comment style